webui.py

```diff
@@ -680,6 +681,11 @@ def flag(self, flag_data, flag_option=None, flag_index=None, username=None):
 
         print("Logged:", filenames[0])
 
+if opt.defaults is not None and os.path.isfile(opt.defaults):
+    with open(opt.defaults, "r", encoding="utf8") as f:
+        user_defaults = yaml.safe_load(f)
+else:
+    user_defaults = {}

```

Comment:
It shouldn't really ever reach this, but pebkac again, if the user specifies a path that is not a file it will